ladybug_rhino/bakegeometry.py

```diff
@@ -234,23 +257,25 @@ def _get_attributes(layer_name=None, attributes=None):
 
 def _get_layer(layer_name):
     """Get a layer index from the Rhino document from the layer name."""
+    doc = rhdoc.ActiveDoc
     layer_table = doc.Layers  # layer table
-    layer_index = layer_table.FindByFullPath(layer_name, True)
-    if layer_index < 0:
+    layer_index = layer_table.FindByFullPath(layer_name, UnsetIntIndex)
```

Comment:
Did we test the use of this RhinoCommon method in Rhino 6? I'm just always wary of these RhinoCommon changes breaking things for some of our users with the old version and we may need a Rhino version check here.